layouts/partials/page/script.html

```diff
@@ -42,8 +42,9 @@
 <script src="{{ $.Site.Params.enterprise_script }}"></script>
 <script>
     const toolbarHeight = $("#presidium-enterprise").height()
+
     if(toolbarHeight) {
-      $("#presidium-navigation").css({ top: `${toolbarHeight}px` });
+      $('#presidium-navigation').css({ top: `${toolbarHeight + 4}px`, height: `calc(100vh - ${toolbarHeight + 4}px)` });
```

Comment:
We can't change the height here because it won't be backward compatible. We also shouldn't be hardcoding offsets. Can we keep the toolbar height the same?